modules/mobile-agents/android/jax-rs/src/main/java/org/wso2/carbon/mdm/services/android/bean/Encrypt.java

```diff
@@ -23,7 +23,7 @@
 /**
  * Bean class of Encrypt-storage operation.
  */
-public class EncryptBean {
+public class Encrypt {
```

Comment:
Encrypt is usually a "verb". Maybe we can use something like "EncryptionData/EncryptionInfo"